plugins/spamassassin.js

```diff
@@ -20,11 +24,11 @@ exports.load_spamassassin_ini = function () {
         max_size:     500000,
         old_headers_action: "rename",
         subject_prefix: "*** SPAM ***",
-        bypass_extra_headers: false,
+        add_headers: true,
     };
 
     for (var key in defaults) {
-        if (plugin.cfg.main[key]) continue;
+        if (plugin.cfg.main[key] !== undefined) continue;
         plugin.cfg.main[key] = defaults[key];
```

Comment:
it makes sense once there's no "add_headers" key in "defaults".

the original code fails if the key is defined under defaults and configuration explicitly sets it to false... "defaults" value will always prevail. This change was to prevent this behaviour.